orcid-message-listener/src/main/java/org/orcid/listener/ReIndexListener.java

```diff
@@ -98,26 +98,29 @@ public void processMessage(final Map<String, String> map) throws JsonProcessingE
             }
             return;
         }
-                
-        //Fetch 2.0 record
-        try {
-            record = orcid20ApiClient.fetchPublicProfile(orcid);
-        } catch(Exception e) {
-            LOG.warn("Unable to fetch record " + orcid + " from 2.0 API");            
-        }
-        
+             
         if(profile != null) {
             //Update solr
             if(isSolrIndexingEnalbed) {
                 solrIndexUpdater.updateSolrIndex(record,profile.toString());
```

Comment:
This is now going to have horrible null pointer exceptions.  SOLR indexing requires 1.2 and 2.0.